Sources/Core/Presenter/UploadingItem.swift

```diff
@@ -185,10 +185,12 @@ public final class UploadingItem: Equatable {
     }
     
     public static func == (lhs: UploadingItem, rhs: UploadingItem) -> Bool {
-        lhs.url == rhs.url
+        lhs.channel == rhs.channel
+            && lhs.url == rhs.url
```

Comment:
It's changing logic, right?

Reply:
I think we forgot to update it. Nothing should be changed.